abi/abi.go

```diff
@@ -362,6 +366,25 @@ func checkReportSizes(r *pb.Report) error {
 	return nil
 }
 
+// ValidateReportFormat returns an error if the provided buffer violates structural expectations of
+// attestation report data.
+func ValidateReportFormat(r []byte) error {
+	if len(r) < ReportSize {
+		return fmt.Errorf("report size is %d bytes, expected %d bytes", len(r), ReportSize)
+	}
+
+	version := binary.LittleEndian.Uint32(r[0x00:0x04])
+	if version != ExpectedReportVersion {
+		return fmt.Errorf("report version is: %d. Expected %d", version, ExpectedReportVersion)
```

Comment:
super nit: Above, on line #373, we're doing "[...], expected [...]". Here, we're doing "[...]. Expected [...]". We should be consistent.

Reply:
Done.